zilliqa/src/consensus.rs

```diff
@@ -3208,6 +3221,21 @@ impl Consensus {
     pub fn get_sync_data(&self) -> Result<Option<SyncingStruct>> {
         self.sync.get_sync_data()
     }
+
+    /// This function is intended for use only by admin_forceSetView API. It is dangerous and should not be touched outside of testing or test network recovery.
+    ///
+    /// Force set our view and override exponential timeout value with given value for this view only
+    /// View value must be larger than current
+    pub fn force_set_view(&mut self, view: u64, timeout_mins: u64) -> Result<()> {
+        if self.get_view()? >= view {
```

Comment:
nit 
```suggestion
        if self.get_view()? > view {
```

if `view == self.get_view()?` everything is fine and the API is just saying "stay in this view for the time I'm telling you, not the time you've calculated".

Reply:
true